include/fastdds/dds/xtypes/dynamic_types/DynamicTypeBuilder.hpp

```diff
@@ -165,6 +165,7 @@ class DynamicTypeBuilder : public std::enable_shared_from_this<DynamicTypeBuilde
 
     /*!
      * Apply the given annotation to a member of this type.
+     * @param[in] member_id the id of the member.
```

Comment:
```suggestion
     * @param[in] member_id Member identifier.
```

Reply:
Fixed in [8020bf2](https://github.com/eProsima/Fast-DDS/pull/4240/commits/8020bf2136b0ed793a5e093b26fc59595dbd7b26)